compositions/NFTPage/NFTInfoSidebar.tsx

```diff
@@ -25,7 +25,7 @@ export function NFTInfoSidebar({ ...props }: NFTInfoSidebar) {
   }, [activeAuctionId, tokenId])
 
   const hasPrevoiusNft = useMemo(() => {
```

Comment:
should be hasPreviousNFT ( typo on your fixed typo commit :) )

Reply:
Couple of things here.
1) According to Matt, sometimes collection tokenId starts with 1 and sometimes with zero
2) We need to know this so the button can be disabled. It's currently possible to go to -1:

https://www.loom.com/share/489b5841b3214fcc97ffaf49d056de77